reports/models.py

```diff
@@ -1 +1,21 @@
-# Create your models here.
+from django.utils.encoding import python_2_unicode_compatible
+from django.db import models
+
+
+@python_2_unicode_compatible
+class ReportTaskStatus(models.Model):
+    """ The status of the report generate tasks
+    """
+
+    start_date = models.CharField(max_length=30, null=False, blank=False)
+    end_date = models.CharField(max_length=30, null=False, blank=False)
+    email_subject = models.CharField(max_length=100, null=False, blank=False)
+    file_size = models.CharField(max_length=30, null=True)
```

Comment:
I think in either place is fine. To get it on this view, you just need to add a `get_file_size` function to your serializer.